app/webpacker/styles/application.scss

```diff
@@ -21,6 +21,7 @@ $govuk-image-url-function: frontend-image-url;
 @import "layout-eyfs/landing-page.scss";
 
 @import "_content-pages-edit.scss";
+@import "_content-assets-edit.scss";
 
 #content_asset_avatar, #content-asset-wrapper {
   padding-top: govuk-spacing(6);
```

Comment:
I thought we had removed mentions of avatar ?

Its not an avatar, its an uploaded image

Please can you rename it ?

Reply:
removed unused css